mrmustard/utils/typing.py

```diff
@@ -49,6 +50,8 @@
 Z = TypeVar("Z", np.int16, np.int32, np.int64)
 N = TypeVar("N", np.uint16, np.uint32, np.uint64)
 
+Mode = int
```

Comment:
I would do the same, but consider that someone may be fooled into thinking that `Mode` is some class that we wrote

Reply:
True.. Would you just leave `int`? 
I feel like it's maybe too generic. I find the type-hints with `int` very confusing, which is why I added `Mode`. Open to change it though